js/models/employee.ts

```diff
@@ -8,6 +8,14 @@ export const Employee = z.object({
   badge_serials: z.array(z.string()),
 });
 
+export const fullName = (emp: Employee | null) => {
+  if (emp === null) {
+    return null;
+  }
```

Comment:
**suggestion (non-blocking):** for utility functions like this, I think that I'd generally prefer that the caller handle things like null cases, but I'll admit I haven't thought this through very rigorously.

Reply:
(same as above) c959b6e